apollo-router/src/plugins/telemetry/mod.rs

```diff
@@ -707,8 +748,9 @@ impl Telemetry {
                 .unwrap_or_else(|| HeaderValue::from_static(""));
             let span = info_span!(
                 ROUTER_SPAN_NAME,
-                query = query.as_str(),
-                operation_name = operation_name.as_str(),
+                graphql.document = query.as_str(),
+                // TODO add graphql.operation.type
```

Comment:
Does this mean "query | mutation | subscription" ?
also: TODO in a future PR or ...